tools/runtime_configs/php/8.1.13

```diff
@@ -0,0 +1,18 @@
+configure_option "--with-curl"
```

Comment:
This file is from the Ubuntu Standard 6.0 image (latest) that Codebuild provides.
https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/standard/6.0/tools/runtime_configs/php/8.1.13